lmiller1990/design-patterns-for-vuejs-source-code
# chore: Update deps

- Vite 2
- Latest Vue (3.1.5)
- Latest testing library/vue
- Use `npm` instead of `yarn`

TODO:

- update book to ensure examples are still working as expected.
